targets/psa_switch/psa_switch.cpp

```diff
@@ -308,55 +308,55 @@ PsaSwitch::ingress_thread() {
     input_buffer.pop_back(&packet);
     if (packet == nullptr) break;
 
-    Parser *parser = this->get_parser("ingress_parser");
-    Pipeline *ingress_mau = this->get_pipeline("ingress");
-    phv = packet->get_phv();
-
     port_t ingress_port = packet->get_ingress_port();
     (void) ingress_port;
     BMLOG_DEBUG_PKT(*packet, "Processing packet received on port {}",
                     ingress_port);
 
+    phv = packet->get_phv();
+
+    Parser *parser = this->get_parser("ingress_parser");
     parser->parse(packet.get());
+
+    // set default std metadata values
+    phv->get_field("psa_ingress_output_metadata.drop").set(true);
```

Comment:
you use a boolean here but below you do a `get_int` which is inconsistent

Reply:
fixed